tests/test_update_sensor.py

```
@@ -162,41 +161,810 @@ async def test_do_update_flow_variants(
+    async def log_update_start(_: str) -> None:
+        call_order.append("log_update_start")
+
+    async def get_current_time() -> datetime:
+        call_order.append("get_current_time")
+        return now
+
+    async def check_device_tracker() -> UpdateStatus:
+        call_order.append("check_device_tracker_and_update_coords")
+        return UpdateStatus.PROCEED
+
+    async def determine_update_criteria(force: bool = False) -> UpdateStatus:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)